providers/src/airflow/providers/edge/executors/edge_executor.py

```diff
@@ -86,9 +88,32 @@ def execute_async(
             )
         )
 
-    @provide_session
-    def sync(self, session: Session = NEW_SESSION) -> None:
-        """Sync will get called periodically by the heartbeat method."""
+    def _check_alive_worker(self, session: Session) -> bool:
```

Comment:
Naming suggestion: `worker liveness`?
```suggestion
    def _check_worker_liveness(self, session: Session) -> bool:
```

Reply:
done